codesign/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Files and directories created by pub.
+.dart_tool/
+.packages
+
+# Conventional directory for build output.
+build/
```

Comment:
Let's split this into several smaller PRs.

For example, my initial thoughts is codesign isn't hooked up into the test logic, so Linux Cocoon isn't running it. We can discuss that more in the initial codesign project creation PR with no extra code added.